source/braille.py

```diff
@@ -792,7 +792,19 @@ def getControlFieldBraille(info, field, ancestors, reportStart, formatConfig):
 		return (_("%s end") %
 			getBrailleTextForProperties(role=role))
 
-def getFormatFieldBraille(field, isAtStart, formatConfig):
+def getFormatFieldBraille(field, fieldCache, isAtStart, formatConfig):

```

Comment:
This breaks backwards compatibility because a new parameter is added in the second position. However, this looked more logical than adding it to the end, because `field` and `fieldCache` are closely related.